UpnetixSystemAlertQueue/Classes/AlertFactoryManager.swift

```diff
@@ -139,3 +148,13 @@ open class AlertFactoryManager {
     }
     
 }
+
+public struct TextFieldAttributes {
```

Comment:
Add documentation for this struct and its' properties.

Reply:
Fixed